docs/apache-airflow/dag-run.rst

```diff
@@ -54,17 +54,31 @@ Cron Presets
 Your DAG will be instantiated for each schedule along with a corresponding
 DAG Run entry in the database backend.
 
-.. note::
+Data Interval
+-------------
+
+Each DAG run in Airflow has an assigned "data interval" that represents the time
+range it operates in. For a DAG scheduled with ``@daily``, for example, each of
+its data interval would start at midnight of each day, and end at midnight of
+the next day.
+
+A DAG run happens *after* its associated data interval has ended, to ensure the
```

Comment:
```suggestion
A DAG run created *after* its associated data interval has ended, to ensure the
```

Reply:
```suggestion
A DAG run is created *after* its associated data interval has ended, to ensure the
```

"happens" is fine, otherwise it should be "is created". The latter might be a better choice, but really either work.

Reply:
I was trying to avoid the passive “is created” because the data interval’s end does not technically trigger the run’s creation, but merely triggers a request to create a run. I’m not sure whether this distinction is meaningful to the user (from an end user’s perspective, this means the run won’t always be created immediately after the data interval ends, but only sometime later). I switched to use “is scheduled” here instead, but other suggestions are still welcomed.